.github/workflows/create-release.yml

```diff
@@ -0,0 +1,11 @@
+name: Create Release
+
+on: [workflow_dispatch]
+
+jobs:
+  tag-and-publish:
+    uses: newrelic/node-newrelic/.github/workflows/release-creation.yml@main
+    with:
+      changelog_file: NEWS.md
```

Comment:
can we change this to CHANGELOG.md and update references to NEWS.md?

Reply:
Sure--no problem there.